docs/csharp/language-reference/operators/division-operator.md

```diff
@@ -16,12 +16,14 @@ author: "BillWagner"
 ms.author: "wiwagn"
 ---
 # / Operator (C# Reference)
-The division operator (`/`) divides its first operand by its second operand. All numeric types have predefined division operators.  
+The division operator (`/`) divides its first operand by its second operand. All numeric types have predefined division operators.
   
 ## Remarks  
  User-defined types can overload the `/` operator (see [operator](../../../csharp/language-reference/keywords/operator.md)). An overload of the `/` operator implicitly overloads the [/= operator](division-assignment-operator.md).  
   
- When you divide two integers, the result is always an integer. For example, the result of 7 / 3 is 2. To determine the remainder of 7 / 3, use the remainder operator ([%](../../../csharp/language-reference/operators/modulus-operator.md)). To obtain a quotient as a rational number or fraction, give the dividend or divisor type `float` or type `double`. You can assign the type implicitly if you express the dividend or divisor as a decimal by putting a digit to the right side of the decimal point, as the following example shows.  
+ When you divide two integers, the result is always an integer. For example, the result of 7 / 3 is 2. To determine the remainde, use the [remainder operator](../../../csharp/language-reference/operators/remainder-operator.md) `%`. For integer types, the `/` operator rounds towards zero. This is not to be confused with floored division, which C# does not have. 
+ 
+ To obtain a quotient as a rational number or fraction, give the dividend or divisor type `float` or type `double`. You can assign the type implicitly if you express the dividend or divisor as a decimal by putting a digit to the right side of the decimal point, as the following example shows.  
```

Comment:
I'm not sure *give type* or *assign the type* are widely used phrasings. The word *cast* should be used. 
*...if you express the dividend or divisor as a decimal...* - that might be confused with `System.Decimal`. Floating-point number?

Reply:
I didn't even notice that, a strange sentence indeed. That was there before my commit. Still, I can correct it.

Reply:
Oh, indeed, sorry, that sentence was already in the text. Anyway, would be nice, if we can make it better.